build/mocha-config.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-    unitTestTimeout: 5000,
+    unitTestTimeout: 3000,
```

Comment:
I had to bump this up due to some timeout issues on one of the build tools (forget which). Honestly it should be like.. 50ms but just need to keep an eye on it if things start failing again